install/kubernetes/cilium/templates/hubble-ui-ingress.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if and (.Values.hubble.ui.enabled) (.Values.hubble.ui.ingress.enabled) -}}
+apiVersion: {{ template "hubble-ui.ingress.apiVersion" . }}
+kind: Ingress
+metadata:
+  name: hubble-ui
+  namespace: {{ .Release.Namespace }}
+  labels:
+    k8s-app: hubble-ui
+{{- with .Values.hubble.ui.ingress.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.hubble.ui.ingress.tls }}
+  tls:
+{{ toYaml .Values.hubble.ui.ingress.tls | indent 4 }}
+{{- end }}
+  rules:
+  {{- range .Values.hubble.ui.ingress.hosts }}
+    - host: {{ . }}
+      http:
+        paths:
+          - path: {{ .Values.hubble.ui.ingress.path }}
```

Comment:
.Values.hubble.ui.ingress.path seems not definied in values.yaml

Reply:
FWIW: I think we should remove this helm value and hardcode the path to `/` for now. Apparently, setting it to anything else than `/` is currently broken and probably to need changes to Hubble UI's fronted code to work (ref #13182)

Reply:
Thanks, makes sense to have it removed :+1: